### PR TITLE
Add option to enable collect

### PR DESCRIPTION
**Pull request by abhinavkgrd**
### commented out the new component, merging only to unblock public-upload

- refactored existing options
- added prop types 
- added enable public collection option

public link manage -> public collect option
image

confirmation before enabling public collect
image

## Test Plan

network request are succeeding and the enableCollect status is correctly updated

### CODE REVIEW

**Comment by abhinavkgrd** [comment]
commented on the code for the time being, until the UI is redesign , to unblock merging public upload

**Review by ua741 — APPROVED** [review]

**Pull request merged**
